### PR TITLE
Remove licensing and telemetry

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Removing licensing and telemetry as they are no longer relevant potentially. This will need a migration to drop the buckets from Bolt before merging

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI

### CODE REVIEW

File: central/license/service/service_impl.go [hunk 1]
```diff
@@ -3,53 +3,15 @@ package service
 import (
```
Comment (viswajithiii) [inline central/license/service/service_impl.go:3]: Should we put a deprecation notice for these APIs so we can remove them eventually?

File: central/telemetry/service/service_impl.go [hunk 2]
```diff
@@ -5,7 +5,6 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
```
Comment (viswajithiii) [inline central/telemetry/service/service_impl.go:6]: Same

**Review by viswajithiii — APPROVED** [review]
image

File: central/license/service/service_impl.go (hunk 1 above)
Comment (connorgorman) [reply central/license/service/service_impl.go:3]: Yes, will do this

**Pull request merged**